packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md

```diff
@@ -183,6 +208,87 @@ The available asides are:
 * `info`
 * `labs`
 
+## Code Checking
+
+To enable code checking for a step, add the `checkable: true` flag to the step in `tutorial.json`.
+
+Next create a file which you will run your code checking. In this example we will call it `_check-code.js`.
+
+> ⚠️ **N.B.** you **MUST** use a `.js` file extension or else this repo will not be able to TS build because we use playground-elements to inject the hidden `_check-code-helpers.js` file.
+
+Then add this code check file as a `hidden` file to the `project.json` of the before directory.
+
+Additionally make sure your `project.json` extends from `/samples/checkable-tutorial-base.json` to include the hidden `_check-code-helpers.js` communication file.
+
+<details open>
+  <summary>example: <code>/before/project.json</code></summary>
+
+```json
+{
+  "extends": "/samples/checkable-tutorial-base.json",
+  "files": {
+    "index.html": {},
+    "my-element.ts": {},
+    "_check-code.js": {"hidden": true}
+  }
+}
+```
+</details>
+
+Next, import this new file into your `index.html` file. And feel free to add the `<!-- playground-fold(-end) -->` comments to hide the import.
+
+
+<details open>
+  <summary>example: <code>index.html</code></summary>
+
+```html
+<head>
+  <!-- playground-fold --><script type="module" src="./_check-code.js"></script><!-- playground-fold-end -->
```

Comment:
Could also recommend using `playground-hide` if we want this to never appear and confuse the user. I think with fold it's possible to expand this import.

Reply:
TIL 💯